rider/src/main/resources/META-INF/plugin.xml

```diff
@@ -48,5 +50,16 @@
               icon="AllIcons.Actions.Collapseall"
               use-shortcut-of="CollapseAll" />
     </group>
+
+    <!-- tool installation -->
+    <action id="Cake.Install.DotNetTool"
+            class="net.cakebuild.actions.CakeToolInstallAction$DotNetToolInstallAction"
+            text="Install .NET Tool" />
```

Comment:
"Install .NET Tool (Global)" or "Install Cake .NET Tool (Global)" to match VS Code?

![image](https://user-images.githubusercontent.com/177608/106084648-1b86b880-60f5-11eb-9e70-993203e8e9ff.png)

Reply:
changed.